types-tests/options.test-d.ts

```diff
@@ -0,0 +1,38 @@
+import { expectType, expectNotType } from 'tsd';
+import { App, OptionsRequest, OptionsSource } from '../dist';
+
+const app = new App({ token: 'TOKEN', signingSecret: 'Signing Secret' });
+
+expectType<void>(app.options('action-id-or-callback-id', async ({ options }) => {
+  expectType<OptionsRequest<OptionsSource>>(options);
+  await Promise.resolve(options);
+}));
+
+// block_suggestion
+expectType<void>(app.options<'block_suggestion'>({ action_id: 'a' }, async ({ options }) => {
```

Comment:
Only `app.options` has the string type parameter like this. `app.view` and `app.action` do not support the same way while they can determine payload type in accordance with `type` in a constraint object.